br_nfe/models/invoice_eletronic.py

```diff
@@ -373,16 +382,18 @@ def _prepare_eletronic_invoice_item(self, item, invoice):
             },
         }
         if item.tipo_produto == 'service':
-            retencoes = item.pis_valor_retencao + \
-                item.cofins_valor_retencao + item.inss_valor_retencao + \
```

Comment:
Evita modificar coisas desnecessárias pois me ajuda depois ao migrar as mudanças pra versão 12, evita conflitos de merge.
E também não introduz bugs desnecessários.

Reply:
ah... só dei uma melhorada no visual, nada de mais... não mudei nada que vai prejudicar

Reply:
São mais para não gerar advertências no pycharm mesmo, pois aquilo me dar tick nervoso e outra que é padrões de código mesmo

Reply:
Só ficou confuso a seguinte situação: Quando uma nota for transmitida, quando ela vai ativar o modo de contingencia ou não ?

Na minha opinião poderia ser o seguinte... 
1. Validar
    * Faz validações padrões ja existente
    * Gera o XML
    * Passa ele no esquema de validação
2. Transmissão
    * Testa para ver se o servidor da sefaz esta funcionando (Consulta status)
        - Gera o xml e Tenta transmitir caso esteja tudo certo
        - Gera o xml com os parâmetros de contingência
        - Gera pdf de modelo de contingencia para impressão
        - Coloca em uma fila especial para fica verificando o servidor da sefaz esta ativo, quando estive tudo certo, faz a transmissão